src/main/java/net/neoforged/neoforge/common/extensions/IBlockExtension.java

```diff
@@ -852,7 +852,7 @@ default boolean canConnectRedstone(BlockState state, BlockGetter level, BlockPos
      * As such, if you need any data from your {@link BlockEntity}, you should put it in {@link ModelData} to guarantee
      * safe concurrent access to it on the client.<br/>
      * {@link ModelDataManager#getAt(BlockPos)} will return the {@link ModelData} for the queried block,
```

Comment:
All references to `ModelDataManager` on this extension method and the appearance one should be replaced with references to the new model data accessor.